matching/README.md

```diff
@@ -137,14 +176,21 @@ optionally a boolean condition Cond, we compute the initial matrix as follows:
 P starts with one column and n rows. P_i1 = LHS[i]. o_1 = []. s_1 = s.
 C_i = (a_i = i, S_i = empty, Cond_i = Cond[i], R_i = []).
 
+Note that while 
+
```

Comment:
Trailing sentence?

Reply:
I think this was a fragment I started writing relating to occurrences and what o_1 meant, but then I ended up adding the relevant text to the section on matrices instead.